src/GovUk.Education.ExploreEducationStatistics.Data.Services/TableBuilderService.cs

```diff
@@ -143,6 +142,14 @@ private async Task<int> GetMaximumTableCellCount(ObservationQueryContext queryCo
             );
         }
 
+        private async Task<Either<ActionResult, Guid>> FindLatestReleaseId(Guid subjectId)
```

Comment:
Would you consider renaming this to `FindLatestPublishedReleaseId` to avoid confusion around whether it's trying to find the latest published versus the latest being worked upon in Admin?  Given that this is in a common Services project, it could potentially be getting used by either.

Reply:
Sure, changed!